src/inet/tests/TestInetCommonPosix.cpp

```diff
@@ -121,7 +121,9 @@ static std::vector<TapInterface> sTapIFs;
 static std::vector<struct netif> sNetIFs; // interface to filter
 
 static bool NetworkIsReady();
+#if !defined(CHIP_DEVICE_LAYER_TARGET_OPEN_IOT_SDK)
```

Comment:
Can this be given a meaningful config value?  What is it about TARGET_OPEN_IOT_SDK that makes this part special?

Reply:
My point was: is there a way to represent the "there is special LwIP init" bit as a platform-independent define that would be set on the Open IoT SDK platform?

Reply:
Do you mean adding e.g. `CHIP_SYSTEM_CONFIG_LWIP_NOT_INIT` define?

Reply:
Done